.github/workflows/tests.yml

```diff
@@ -103,6 +103,9 @@ jobs:
       - uses: nanasess/setup-chromedriver@v1.0.1
       - run: npm install
         name: Add dependencies
+      - run: npm i sass-embedded@~1.57.1
```

Comment:
Please add a TODO to remove once fixed

Reply:
The problem has been solved with a new release of sass-embedded, i'm reverting the changes